src/org/labkey/test/tests/GpatAssayTest.java

```diff
@@ -128,16 +127,49 @@ public void testSteps()
 
         log("Import XLSX GPAT assay");
         assayDesignerPage = startCreateGpatAssay(GPAT_ASSAY_XLSX, ASSAY_NAME_XLSX);
-        setAssayResultsProperties(assayDesignerPage, 9);
+        setAssayResultsProperties(assayDesignerPage, 11);
+
+        DomainFormPanel domainFormPanel = assayDesignerPage.expandFieldsPanel("Run Fields");
+        domainFormPanel.addField(new FieldDefinition("Date", FieldDefinition.ColumnType.Date));
```

Comment:
Is there coverage for converting assay run fields from datetime -> date/time?

Reply:
There is now.